web_console/dynamic-plug-in/dynamic-plug-in.adoc

```diff
@@ -40,4 +38,12 @@ conster Header: React.FC = () => {
 };
 ----
 
-* _Do not_ use console CSS classes in your markup or override console CSS classes. These are not APIs and are subject to change. Using them might break your plug-in. Avoid selectors like element selectors that could affect markup outside of your plug-in’s components.
+* _Do not_ use console CSS classes in your markup or override console CSS classes. These are not APIs and are subject to change. Using them might break your plug-in. Avoid selectors like element selectors that could affect markup outside of your plug-ins components.
```

Comment:
IBM Style Guide: Do not use like to mean "such as."  Maybe:

Avoid selectors, such as element selectors, that... 
Avoid selectors that could affect markup outside of your plug-ins components, such as element selectors.

Reply:
I did a quick search on _do not_. I didn't see any place other than the two instances in the plug-ins doc that uses emphasis on _do not_. I leave it up to you.